scripts/brp-remove-libtool-files

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# If using normal root, avoid changing anything.
+if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
+	exit 0
+fi
+
+find "$RPM_BUILD_ROOT" \( -type f -o -type l \) -name \*.la  -print0 |
+xargs  --no-run-if-empty -0 file -N -L |
+grep "libtool library file" |
+sed -e 's#: libtool library file.*##g' |
```

Comment:
For the brp-check-elf-files script, it's hard to contest that more work has been put into yours than pixel (I'm guessing) put into writing the original perl script, which I've simply rewritten in bash.

Yet for the brp- scripts, I'm a firm believer in KISS, where I really think the brp* scripts should be self-contained and for single purpose only with as little complexity as possible.